.env

```diff
@@ -4,4 +4,8 @@
 # - `rococo_sepolia` for Rococo <=> Sepolia bridge.
 # - `kusama_mainnet` for Kusama <=> Ethereum bridge. (TBD)
 # - `polkadot_mainnet` for Polkadot <=> Ethereum bridge.
-NEXT_PUBLIC_SNOWBRIDGE_ENV=local_e2e
+NEXT_PUBLIC_SNOWBRIDGE_ENV=paseo_sepolia
```

Comment:
Revert this file and lets keep it as the default env. Move all these keys to your env.local as the README suggests.

https://github.com/Snowfork/snowbridge-app/blob/main/README.md?plain=1#L23